azext_edge/edge/providers/support/mq.py

```diff
@@ -92,7 +96,12 @@ def fetch_deployments():
     )
```

Comment:
will double check again

Reply:
Hi @digimaun, I checked that seems like there is only **app.kubernetes.io/managed-by=Helm** on this resource before proposed label change(and since this change will be reverted for apr release), therefore would it be safer to still use field selector?